### PR TITLE
Refactor laser devices

### PR DESCRIPTION
**Pull request by WalterMwaniki**
- This Branch includes updates to the laser device-type base class as well as updates to individual device drivers.   
- The base class for each device type inherits from a base VoxelDevice class.   
Within the `voxel/devices/lasers/` directory, I have included directories that group laser devices by manufacturer.   
- The device-type directory (`voxel/devices/lasers/`) is a package that exports all voxel supported devices.

### CODE REVIEW

**Comment by micahwoodard** [comment]
I think you branched #50 from this branch so I've already reviewed changes in that branch.

**Comment by adamkglaser** [comment]
@WalterMwaniki sorry I have not gotten to this. Will take a look today.

File: voxel/devices/lasers/__init__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+"""
+Available laser devices:
+- voxel.devices.lasers.simulated
+    - SimulatedLaser
+- voxel.devices.lasers.vortran
+    - StradusLaser
+- voxel.devices.lasers.cobolt
+    - SkyraLaser
+- voxel.devices.lasers.oxxius
+    -  OxxiusLBXLaser
```
Comment (adamkglaser) [inline voxel/devices/lasers/__init__.py:10]: one too many spaces in these two

**Review by adamkglaser — APPROVED** [review]
This all looks great to me!

**Comment by adamkglaser** [comment]
@WalterMwaniki this one is OK to merge now.

**Pull request merged**